app/views/ansible_roles/import.html.erb

```diff
@@ -0,0 +1,47 @@
+<% title _("Changed roles") %>
```

Comment:
Changed Ansible roles, just like the index page - otherwise they could be regular Foreman RBAC roles